### PR TITLE
dbmate 1.11.0

### PR DESCRIPTION
**Pull request by dbmate-releasebot**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

**Comment by amacneil** [comment]
I can't see any reason why this test is failing on 10.15:

```sh
/usr/local/Cellar/dbmate/1.11.0/bin/dbmate: cannot execute binary file
```

The build is very simple:

```rb
def install
  system "go", "build", "-ldflags", "-s", "-o", bin/"dbmate", "."
end
```

It works perfectly on 10.14 and 11.0, but fails on 10.15. Any ideas?

**Comment by jonchang** [comment]
Yeah, this is probably the Catalina tar bug again: https://github.com/Homebrew/brew/issues/6539

**Comment by amacneil** [comment]
Interesting. I added some debugging lines, and apparently the binary is just "data":

```sh
==> ls -al /usr/local/Cellar/dbmate/1.11.0/bin/dbmate
-r-xr-xr-x  1 brew  staff  10600768 Nov 19 02:04 /usr/local/Cellar/dbmate/1.11.0/bin/dbmate
==> file /usr/local/Cellar/dbmate/1.11.0/bin/dbmate
/usr/local/Cellar/dbmate/1.11.0/bin/dbmate: data
==> /usr/local/Cellar/dbmate/1.11.0/bin/dbmate create
/usr/local/Cellar/dbmate/1.11.0/bin/dbmate: /usr/local/Cellar/dbmate/1.11.0/bin/dbmate: cannot execute binary file
Error: dbmate: failed
```

It sounds like the only workaround is for a homebrew maintainer to manually remove the Catalina bottle?

**Comment by amacneil** [comment]
I believe this is ready to merge despite the CI failure on Catalina. If there is anything I can do to resolve the failure please let me know.

File: Formula/dbmate.rb [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-  sha256 "bf00360c9eb12151c157cc227a21f6688d8ea89de571b618a018b081637297a3"
+  url "https://github.com/amacneil/dbmate/archive/v1.11.0.tar.gz"
+  sha256 "97c017061a6ae1d3f2c614306639990fa0dab656e329a353467f9515ca999bb9"
   license "MIT"
   head "https://github.com/amacneil/dbmate.git"
 
   bottle do
     cellar :any_skip_relocation
     sha256 "1c3359021c62b14a50a3478e5e64a28ea310de8e5cf62c449120b89d4e205baa" => :big_sur
-    sha256 "40c2f2d04a9d033478fdff1a272fa0d125a1dbdd746154afe97c5c36612a36af" => :catalina
```
Comment (fxcoudert) [inline Formula/dbmate.rb:12]: Please do not touch the bottle block

**Review by fxcoudert — CHANGES_REQUESTED** [review]
We will reenable a work-around the Catalina bug, and we can merge once that is done

File: Formula/dbmate.rb (hunk 1 above)
Comment (amacneil) [reply Formula/dbmate.rb]: Removed

**Comment by chenrui333** [comment]
looks like flakiness, rerun a bit.

**Review by fxcoudert — APPROVED** [review]

**Comment by amacneil** [comment]
🙏 thanks for fixing Catalina issue

**Pull request closed without merging**